README.md

```diff
@@ -57,9 +57,9 @@ For in-depth testing guidance review the [contribution guidelines](./CONTRIBUTIN
 ### Official Image Builds
 odh-dashboard images are automatically built and pushed to [quay.io](https://quay.io/repository/opendatahub/odh-dashboard) after every commit to the `main` branch. The image tag name format for each image is `main-<COMMIT SHORT HASH>`.
 
-Example: The `main` branch is updated with commit `f76e3952834f453b1d085e8627f9c17297c2f64c`.  The CI system will automatically build an odh-dashboard image based on that code and push the new image to `odh-dashboard:main-f76e395`
+Example: The `main` branch is updated with commit `f76e3952834f453b1d085e8627f9c17297c2f64c`.  The CI system will automatically build an odh-dashboard image based on that code and push the new image to `odh-dashboard:main-f76e395` and updated `odh-dashboard:main` to point to the same image hash
 
-The [latest](https://quay.io/opendatahub/odh-dashboard:latest) tag is a floating tag that is updated nightly and points to the most recent `main-<HASH>` commit from the previous day.
+The [nightly](https://quay.io/opendatahub/odh-dashboard:nightly) tag is a floating tag that is updated nightly and points to the most recent `main-<HASH>` commit from the previous day.
```

Comment:
Is `latest` just not used anymore?

Reply:
`latest` is not used only because we would need to update the openshift-ci automation to add a `latest` tag at the same time that `main` is updated